### DISCUSSION TITLE
Support Learn Mode to become a secondary controller for Homey etc.

### DISCUSSION THREAD

**Discussion by thomaswitt** [zwave-js/zwave-js#6427] [Feature Requests]
### Description

I bought an Aeotec Z-Stick 7 for my RPi 4 and hoped it could be used as a secondary controller for my Homey Pro system (as the Homey Aeotec App supports the Z-Stick Gen 5 officially). There's even some [official documentation](https://aeotec.freshdesk.com/support/solutions/articles/6000263528-z-stick-gen-5-as-secondary-controller-in-homey) on the Aeotec page for that. I wanted to gain three functionalities by using Z-Wave JS UI with that:

1. Seamless MQTT integration - so that all actions of all my devices can be broadcasted to my MQTT broker.
2. Firmware Updates for Z Wave Devices (as Homey doesn't support Firmware Updates)
3. Getting easy API access to send commands to Z Wave Devices (e.g. turning on/off a light via an API call easily)

Unfortunately, Z-Wave JS UI - as I understand - doesn't support learn mode at the moment and therefore I can't include it as a device/secondary controller in my existing network.

### Do you have a solution in mind?

I would suggest to add support for Z-Wave UI JS for being a secondary controller.

### Describe alternatives you've considered

There are non as I don't want to drop Homey as my primary system.

**Comment by AlCalzone** [comment]
I thought we had an actual issue for this already, but we don't. Escalating to an issue: https://github.com/zwave-js/node-zwave-js/issues/6428
